DEPENDENCIES

```diff
@@ -169,7 +169,7 @@ npm/npmjs/-/cssom/0.3.8, MIT, approved, clearlydefined
 npm/npmjs/-/cssom/0.4.4, MIT, approved, clearlydefined
 npm/npmjs/-/cssstyle/2.3.0, MIT, approved, clearlydefined
 npm/npmjs/-/csstype/3.1.1, MIT, approved, clearlydefined
-npm/npmjs/-/cx-portal-shared-components/1.5.0, , restricted, clearlydefined
```

Comment:
We should wait until the ticket is evaluated before merging this. We cannot create a release with a restricted dependency. If it is really important for some reason to merge this to main, we can do that without creating a release @adityagajbhiye9